README.md

```diff
@@ -36,14 +36,13 @@ Open main.py which contains a simple starter example.
 
 ### Initialization
 
-The `ContractUtility` class is used to compile and deploy the contracts, 
+The ```ContractUtility``` class is used to compile and deploy the contracts, 
```

Comment:
Why would you want to put ContractUtility in a separate line?

```suggestion
The `ContractUtility` class is used to compile and deploy the contracts, 
```

Reply:
Updated ```ContractUtility```